mods/fireflies/init.lua

```diff
@@ -113,14 +118,14 @@ minetest.register_craft( {
 	recipe = {
 		{"farming:string", "farming:string"},
 		{"farming:string", "farming:string"},
-		{"group:stick", ""}
+		{"default:stick", ""}
```

Comment:
Unrelated change, please undo.

Reply:
Obviously, I have some pollution in the fromer original code. Bad boy I am.

THX and will depollute...